gce/util.sh

```diff
@@ -9,6 +9,171 @@
 #	util::remote_cmd
 
 
+
+# Helpers #
+# These functions implement lower level operations required for deploying
+# and destroying Google Compute Engine instances.  These functions should be
+# considered private, scoped to the provider.
+
+function __init_network() {
```

Comment:
Do you want the "__" prefix to mean something special, vs. using "util::" prefix here?

Reply:
Mentioned in a comment above.  My intent was to have break up provider specific code into "private" funcs, denoted by `__` prefix.  As a rule, they would be called only within the provider implementations of the common funcs (create_instances, destroy_instances, etc.)  But I admit this is probably trying to do Go in bash, what do you think?

Reply:
We're not providing a package per se so all funcs in the helper files are callable. For now I do not think it's worth breaking the naming convention to designate lower level funcs...